libr/anal/fcn.c

```diff
@@ -1491,13 +1426,6 @@ static int fcn_recurse(RAnal *anal, RAnalFunction *fcn, ut64 addr, ut8 *buf, ut6
 			}
 			// switch statement
 			if (anal->opt.jmptbl) {
-				if (fcn->refs->tail) {
```

Comment:
@Toizi @radare maybe I should not remove these lines, right? :)
What I don't understand though is... what is the "tail" of the refs? What is the "last" ref?

Reply:
i think fcn->refs->tail is an assumption to check  if there's any ref